hack/build-and-push.sh

```diff
@@ -172,6 +172,8 @@ do
       ANNOTATIONS+=("org.opencontainers.image.source=${VCS_URL}")
       ANNOTATIONS+=("org.opencontainers.image.revision=${VCS_REF}")
       ANNOTATIONS+=("org.opencontainers.image.url=${VCS_URL}/tree/${VCS_REF}/${task_dir}")
+      concrete_task_version=$(yq '.metadata.labels."app.kubernetes.io/version"' "$prepared_task_file")
```

Comment:
Please use just `task_version`

Reply:
Variable `task_version` is already used in the `read` command to reference the version from task file path.